core/domain/interaction_jobs_one_off.py

```diff
@@ -152,6 +152,39 @@ def reduce(key, values):
         yield (key, values)
 
 
+class MultipleItemInteractionLtOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """Audit job that flags MultipleItemInteractions with too many choices.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_fetchers.get_exploration_from_model(item)
+        interactions_to_check = ('MultipleChoiceInput', 'ItemSelectionInput')
+        for state in exploration.states.items():
+            if state[1].interaction.id in interactions_to_check:
+                choices = state[1].interaction.customization_args['choices']
+                for choice in choices.value:
+                    choice_length = len(choice.html)
+                    if choice_length > 30:
+                        yield ('LONGER_THAN_30', (item.id, choice_length))
```

Comment:
Maybe yield state name as well?

Reply:
The purpose of audit job is [link](https://github.com/oppia/oppia/issues/11267#issuecomment-783259128) and I don't think so yeilding state_name is required in this job.